### PR TITLE
DBAAS-120 Remove tabs use single page flow

### PR DESCRIPTION
**Pull request by yzhao583**
Remove tabs use single page flow

### CODE REVIEW

File: src/components/dbaasImportPage.jsx [hunk 1]
```diff
@@ -1,50 +1,42 @@
...
-    };
-
-    setActiveTab(tabIndex) {
-        this.setState({
-            activeTabKey: tabIndex
-        });
-    }
-
     setSelectedDBProvider(dbProvider) {
+        console.log(dbProvider);
```
Comment (jeremyary) [inline src/components/dbaasImportPage.jsx:37]: should we leave the console log here or was it for debugging?

File: src/components/dbaasImportPage.jsx [hunk 2]
```diff
@@ -61,63 +53,34 @@ class DBaasImportPage extends React.Component {
...
                     <div className="section-breadcrumb">
-                        <span className="breadcrumb-link">DBaaS Connect DataBase Account</span>
+                        <span className="breadcrumb-link" onClick={this.goBack}>Database-as-a-Service</span>
                         <span className="breadcrumb-chevron"> > </span>
-                        Database Provider Details
+                        Create Provider Account
                     </div>
-                    <div className="section-title extra-bottom-margin">DBaaS Connect DataBase Account</div>
+                    <div className="section-title">Create Provider Account</div>
+                    <div className="section-subtitle">Creating a Provider Account resource allows provider clound instances to be imported</div>
```
Comment (jeremyary) [inline src/components/dbaasImportPage.jsx:65]: misspell of 'cloud' ('clound') here

**Review by jeremyary — CHANGES_REQUESTED** [review]

File: src/components/dbaasImportPage.jsx (hunk 1 above)
Comment (yzhao583) [reply src/components/dbaasImportPage.jsx]: Thanks! removed!

File: src/components/dbaasImportPage.jsx (hunk 2 above)
Comment (yzhao583) [reply src/components/dbaasImportPage.jsx]: Thanks! Fixed!

**Review by jeremyary — APPROVED** [review]

**Pull request merged**